src/recognition/ErrorFunction.java

```diff
@@ -0,0 +1,232 @@
+/**
+ *
+ * @file
+ * @brief Here is Error function for checking accuracy of Classifier work
+ * Also here implemented RandomClassifier which assigns random image to class from
+ * zero  to nine
+ *
+ *
+ * @authors Vladislav Shikhanov
+ *
+ *		  http://move-llc.ru
+ ************************************************************************************/
+package recognition;
+
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Random;
+
+import static recognition.ReadIDX.ImageIDX;
+import static recognition.ReadIDX.testCol;
+
+class ErrorFunction {
+    /*********************** Collections with images definition*********************/
+    private ReadIDX readIDX = new ReadIDX();
+    private ImagesByClass imagesByClass = new ImagesByClass();
+
+    /********************************************************************************
+     *
+     * accordanceAnalyse Method checks all collections with drawings to accordance
+     * assigned images to real values
+     *
+     * @param imagesByClass is compilation of collection of numbers (collection with
+     *                      drawings of 0, of 1 etc)
+     *******************************************************************************/
+    void accordanceAnalyse(ImagesByClass imagesByClass) {
+        for ( int i = 0; i < 10; i++) {
```

Comment:
I see one extra space here. 
The statement
for (int i = 0; ...) is more Java way.

Reply:
Yes, I'll pay more attention to spaces in future and I'll fix it in already existing code. I understand that it's not really good looking, so I try to stick code style in new code. It's bad habbit to use extra spaces as well as missing spaces in other places, I try to control this.
Thank you.